engine/hatchery/main.go

```diff
@@ -225,24 +226,23 @@ func parseConfig(cmd *cobra.Command) HatcheryMode {
 		sdk.Exit("CDS api endpoint not provided. See usage:\n%s\n", cmd.Short)
 	}
 
-	usr := viper.GetString("cds-user")
-	passwd := viper.GetString("cds-password")
+	uk = viper.GetString("token")
+
 	token := os.Getenv("CDS_TOKEN")
```

Comment:
uk = viper.GetString("token")
token := os.Getenv("CDS_TOKEN")

one of those could be removed